src/seq/minhash.jl

```diff
@@ -0,0 +1,118 @@
+# MinHash
+# ========
+#
+# Functions to generate MinHash sketches of biological sequences
+#
+# This file is a part of BioJulia.
+# License is MIT: https://github.com/BioJulia/Bio.jl/blob/master/LICENSE.md
+using DataStructures: SortedSet
+
+"""
+MinHash Sketch type
+
+MinHash sketches are a sorted set of kmer hashes, containing the smallest `s`
+hashes for kmers of a given length. The type contains two parameters:
+
+* .sketch: a sorted set of hashes
+* .kmersize: the length of kmers used to generate the sketch
+"""
+type MinHashSketch
+    sketch::Vector{UInt64}
+    kmersize::Int
```

Comment:
I mean doing something like the following gist: https://gist.github.com/Ward9250/1c550e345b742f3287da4f9a2c2fda08